WordPress/Classes/ViewRelated/NUX/SiteCreationDomainsActivityTableViewCell.xib

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
And this constraint error:
<img width="330" alt="screen shot 2017-12-19 at 3 11 44 pm" src="https://user-images.githubusercontent.com/1816888/34181297-f41e4696-e4ce-11e7-9946-5ffcabf68c7f.png">